code/modules/mod/modules/modules_general.dm

```diff
@@ -633,3 +633,98 @@
 
 /obj/item/mod/module/plasma_stabilizer/on_unequip()
 	REMOVE_TRAIT(mod.wearer, TRAIT_NOSELFIGNITION, MOD_TRAIT)
+
+//Finally, https://pipe.miroware.io/5b52ba1d94357d5d623f74aa/mspfa/Nuke%20Ops/Panels/0648.gif can be real:
+///Hat Stabilizer - Allows displaying a hat over the MOD-helmet, à la plasmamen helmets.
+/obj/item/mod/module/hat_stabilizer
+	name = "MOD hat stabilizer module"
+	desc = "A simple set of deployable stands, directly atop one's head; \
+	these will deploy under a select few hats to keep them from falling off, allowing them to be worn atop the sealed helmet.\
+	\n	You still need to take the hat off your head while the helmet deploys, though.\
+	\n	This is a must-have for Nanotrasen Captains, enabling them to show off their authoritative hat even while in their MODsuit."
+	icon_state = "hat_holder"
+	/*Intentionally left inheriting 0 complexity and removable = TRUE;
```

Comment:
```suggestion
	incompatible_modules = list(/obj/item/mod/module/hat_stabilizer)
	/*Intentionally left inheriting 0 complexity and removable = TRUE;
```
tbh, all mods should just automatically be incompatible with themselves when everything first loads. Would mean not having to do this for every single new module..